MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include license.txt Legal.txt versioneer.py src/hdmf_zarr/_version.py src/hdmf_zarr/_due.py
```

Comment:
Is this case-sensitive? The repo contains `License.txt`. It also does not contain `Legal.txt`. I am not sure if that matters. Can you try creating a wheel and distribution to see if these files get packaged correctly?

Reply:
I believe it's case insensitive. I updated it either way.

Reply:
I did a test with creating a wheel and it is case insensitive.